.gitignore

```diff
@@ -14,5 +14,7 @@ yarn-error.log
 .tmp
 
 /scripts/deployment-test/apps
+/scripts/deployment-test/package-lock.json
+/scripts/deployment-test/yarn.lock
```

Comment:
I don't think we need these to be ignored. I think we want to keep these around.

Reply:
deployment tests get their remix-dev version bumped from the version script, so i think we'd want these ignored to prevent any accidental mismatches

Reply:
Ah, whoops. I brought this back in the PR I just merged. Could someone remove that? I'm going to be away from the keyboard for a few days.

Reply:
Removed it in https://github.com/remix-run/remix/pull/2669